docs/index.md

```diff
@@ -0,0 +1,28 @@
+VFS for Git: Virtualized File System for Git
+============================================
+
+VFS stands for Virtual File System. VFS for Git virtualizes the file system
+beneath your Git repo so that Git and all tools see what appears to be a
+normal repo, but VFS for Git only downloads objects as they are needed.
+VFS for Git also manages the files that git will consider, to ensure that

```

Comment:
```suggestion
VFS for Git also manages the files that Git will consider, to ensure that

```

Reply:
Thanks! This is leftover from the readme, but I tried to check the rest of the docs, too.